.eslintrc.js

```diff
@@ -1,20 +1,48 @@
-'use strict';
-
 module.exports = {
   root: true,
-  extends: ['eslint:recommended', 'prettier'],
-  plugins: ['prettier'],
   parserOptions: {
     ecmaVersion: 2017,
+    sourceType: 'module'
   },
+  plugins: [
+    'ember'
+  ],
```

Comment:
Prettier is added as a devDep, but isn't configured. Did you intend to configure it?

Reply:
We can configure it in a later PR.